ts/Extensions/Breadcrumbs.ts

```diff
@@ -0,0 +1,1187 @@
+         * @type      {Highcharts.BreadcrumbsClickCallbackFunction}
+         * @since     next
+         * @apioption breadcrumbs.events.click
+         */
+
+        /**
+         * When the breadcrumbs are floating, the plot area will not move
+         * to make space for it. By default, the chart will not make space
+         * for the buttons. This property won't work when positioned in the
+         * middle.
+         *
+         * @sample {highcharts} highcharts/breadcrumbs/floating-false
+         *          Non-floating breadcrumbs
+         *
+         * @type      {boolean}
+         * @since     next
+         */
+        floating: false,
+
+        /**
+         * A format string for the breadcrumbs button.
+         * Variables are enclosed by curly brackets.
+         * Available values are passed in the declared point options.
+         *
+         * @type      {string|undefined}
+         * @since     next
+         * @default   undefined
+         * @sample {highcharts} highcharts/breadcrumbs/format
+         *          Display custom values in breadcrumb button.
+         */
+        format: void 0 as any,
+
+        /**
+         * Callback function to format the breadcrumb text from scratch.
+         *
+         * @type      {Highcharts.BreadcrumbsFormatterCallbackFunction}
+         * @since     next
+         * @apioption breadcrumbs.formatter
+         */
+
+        /**
+         * What box to align the button to. Can be either `plotBox` or
+         * `spacingBox`.
+         *
+         * @type       {Highcharts.ButtonRelativeToValue}
+         * @default    plotBox
+         * @since      3.0.8
+         * @product    highcharts highmaps
+         * @apioption  drilldown.breadcrumbs.relativeTo
+         */
+        relativeTo: 'plotBox',
+
+        /**
+         * Positioning for the button row. The breadcrumbs buttons
+         * will be aligned properly for the default chart layout
+         * (title,  subtitle, legend, range selector) for the custom chart
+         * layout set the position properties.
+         * @sample {highcharts} highcharts/breadcrumbs/position
+         *          Custom button positioning.
+         */
+        position: {
+            /**
+             * Horizontal alignment of the breadcrumbs buttons.
+             *
+             * @type {Highcharts.AlignValue}
+             */
+            align: 'left',
+
+            /**
+             * Vertical alignment of the breadcrumbs buttons.
+             *
+             * @type {Highcharts.VerticalAlignValue}
+             */
+            verticalAlign: 'top',
+
+            /**
+             * The X offset of the breadcrumbs button group.
+             */
+            x: 0,
+
+            /**
+             * The Y offset of the breadcrumbs button group.
+             */
+            y: 0
+        },
+
+        /**
+         * Options object for Breadcrumbs separator.
+         *
+         * @since     next
+         */
+        separator: {
```

Comment:
Is this used at all now? In the demos, there is now an arrow between the items.

Reply:
That's true, with the current version symbol was replaced by the string type arrows. Would you like to have the possibility to choose a symbol as a separator?

Reply:
Actually, I have found your answer from some time ago:

> Oh, this is between the buttons. But I still think we should use a simple character.

So I will just go with removing symbol options and keeping full text support for separators.